perf: Also mention THP in Redis and refactor how we are mentioning THP so far (#1771)
theforeman/foreman-documentation: guides/common/modules/con_redis-tuning.adoc

[id="Redis_Tuning_{context}"]
= Redis Tuning

> Redis is a in-memory data store.

Reviewer: ```suggestion
Redis is an in-memory data store.
```

It is used by multiple services in {Project}.

> Dynflow and Pulp tasking systems use it to track it's tasks.

Reviewer: ```suggestion
Dynflow and Pulp's tasking systems use it to track their tasks.
```

Reviewer 2: Actually, we do not use possive form (**'s**) for products. See https://docs.engineering.redhat.com/display/CCSENABLE/Possessives+and+apostrophes for reference (Red Hat VPN needed).

> Dynflow and Pulp tasking systems use it to track their tasks.

Reviewer: I think an article is needed here, but not 100% sure since I'm not a native speaker.
```suggestion
The Dynflow and Pulp tasking systems use it to track their tasks.
```

> Given the way {Project} uses the Redis, it's memory requirements should be stable.

Reviewer: ```suggestion
Given the way {Project} uses the Redis, its memory consumption should be stable.
```

> Given the way {Project} uses the Redis, its memory consumption should be stable.

Reviewer 3: ```suggestion
Given the way {Project} uses Redis, its memory consumption should be stable.
```

> We reccomend disabling Transparent Huge Pages on server running Redis.

Reviewer: ```suggestion
We recommend disabling Transparent Huge Pages on server running Redis.
```

> The Redis authors recommend disabling Transparent Huge Pages on server running Redis.

Reviewer: ```suggestion
The Redis authors recommend disabling Transparent Huge Pages on servers running Redis.
```
